HotelBookingTests/HotelBookingTest.cs

```diff
@@ -32,9 +35,17 @@ public void ShouldReturnFalseAndCallLoggerWhenHotelIdIsNotAvailable()
         [Test]
         public void ShouldReturnReservationResultForHotelWithCorrectId()
         {
-            var result = _manager.MakeReservation(1, 3.0, 3333, "test@gmail.com", DateTime.Now);
+            int id = 1;
+            var result = _manager.MakeReservation(id, 3.0, 3333, "test@gmail.com", DateTime.Now);
             Assert.IsInstanceOf(typeof(ReservationResult), result);
+            Logger.AssertWasNotCalled(l => l.Write($"Couldn't find hotel with ID: {id}"));
         }
 
+        [Test]
+        public void ShouldReturnAvailableHotelIds()
+        {
+            List<int> listHotels = _manager.PresentAvailableHotels();
+            Assert.AreEqual(listHotels, new List<int> {1, 2});
```

Comment:
To consider.

Could you please make sure that this assert compares items not list reference